src/main/scala/org/bitcoins/core/script/crypto/CryptoInterpreter.scala

```diff
@@ -226,22 +226,22 @@ trait CryptoInterpreter extends ControlOperationsInterpreter with BitcoinSLogger
             }
             logger.debug("nPossibleSignatures: " + nPossibleSignatures)
             val (pubKeysScriptTokens, stackWithoutPubKeys) =
-              (program.stack.tail.slice(0, nPossibleSignatures.num.toInt),
-                program.stack.tail.slice(nPossibleSignatures.num.toInt, program.stack.tail.size))
+              (program.stack.tail.slice(0, nPossibleSignatures.underlying.toInt),
+                program.stack.tail.slice(nPossibleSignatures.underlying.toInt, program.stack.tail.size))
```

Comment:
Let's create a `.toInt` function on `ScriptNumber` instead of calling `underlying.toInt`